pom.xml

```diff
@@ -82,8 +74,11 @@
     <properties>
         <netty.version>4.1.0.CR3</netty.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <jetty.alpn.path>${settings.localRepository}/org/mortbay/jetty/alpn/alpn-boot/${jetty.alpn.version}/alpn-boot-${jetty.alpn.version}.jar</jetty.alpn.path>
-        <argLine.bootcp>-Xbootclasspath/p:${jetty.alpn.path}</argLine.bootcp>
+        <alpn-api.version>1.1.2.v20150522</alpn-api.version>
```

Comment:
I get why we want to define version numbers for things like `alpAgent` in properties, since we use them in several places in here, and we definitely want to be consistent. Would it make more sense to inline `alpn-api.version`, though? It looks like we only use it in one place.

I'm very open to arguments that, since it's sensitive to the version of Netty we're using, the ALPN API version and Netty version should be close to each other for readability.